Lib/test/test_launcher.py

```diff
@@ -623,7 +623,7 @@ def test_search_path(self):
                     [script, "-postarg"],
                     env={"PATH": f"{Path(sys.executable).parent};{os.getenv('PATH')}"},
                 )
-        self.assertEqual(f"{sys.executable} -prearg {script} -postarg", data["stdout"].strip())
+        self.assertEqual(f"X.Y.exe -prearg {script} -postarg", data["stdout"].strip())
```

Comment:
This is a negative impact of the change. There might be a way to avoid it, but I haven't found it today - maybe when I look again next week.